pkg/labels/fake_applicator_test.go

```diff
@@ -2,6 +2,7 @@ package labels
 
```

Comment:
am I understanding correctly that this is testing `WatchMatchDiff()` on the fake applicator but there's no test for `WatchMatchDiff` on the real one?